addons/hw_drivers/websocket_client.py

```diff
@@ -13,6 +18,68 @@
 _logger = logging.getLogger(__name__)
 websocket.enableTrace(True, level=logging.getLevelName(_logger.getEffectiveLevel()))
 
+WS_REQUEST_POOL = concurrent.futures.ThreadPoolExecutor(8)
```

Comment:
Need to check whether 8 is a good amount

Reply:
![image](https://github.com/odoo/odoo/assets/36443074/fe3e807c-53ba-4252-844d-b0663d256a23)